libs/model/src/quest/CreateQuest.command.ts

```diff
@@ -0,0 +1,35 @@
+import { Command } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { models } from '../database';
+import { AuthContext, isAuthorized } from '../middleware';
+import { mustNotExist } from '../middleware/guards';
+
+export function CreateQuest(): Command<
+  typeof schemas.CreateQuest,
+  AuthContext
+> {
+  return {
+    ...schemas.CreateQuest,
+    auth: [isAuthorized({ roles: ['admin'] })],
+    secure: true,
+    body: async ({ payload }) => {
+      const { community_id, name, description, start_date, end_date } = payload;
+
+      const existingName = await models.Quest.findOne({
+        where: { community_id, name },
```

Comment:
nit but we can just pull id from this like:

```typescript
const existingName = await models.Quest.findOne({
        where: { community_id, name },
        attributes: ['id'],
});
```

This would convert this from a select * to a select id and pull less info.